api/README.md

```diff
@@ -31,10 +31,11 @@ Some environment variables are pre-configured for the development. You can copy
 
 - `DATABASE_URL` (required): path to the database file
 - `JWT_SECRET` (required): server's jwt secret
-- `PORT` (optional, default: 3000): port on which the server will run
-- `AUTH_SIGNUP_ENABLED` (optional, default: `false`): if signing up mutation is allowed (i.e. user creation via endpoint is enabled)
-- `JWT_EXPIRATION_PERIOD` (optional, default: `'7d'`): how soon the signed jwt token will expire
-- `DEBUG` (optional): if set, enables the different more explicit logging mode where debug levels are set to `debug` for the app's logger and `query` for db logger
+- `PORT` (optional, default `3000`): port on which the server will run
+- `API_ORIGIN` (optional, default `http://0.0.0.0:${PORT}`): the URL at which the API is running. it's important to provide this variable in production since it influences the message signed during authorization
+- `AUTH_SIGNUP_ENABLED` (optional, default: `false`): if signing up is allowed. In case it's not set, no new users can be created, but old users can still sign up
```

Comment:
nit
```suggestion
- `AUTH_SIGNUP_ENABLED` (optional, default: `false`): if signing up is allowed. In case it's not set, new users _cannot_ be created, but old users _can_ still sign up
```